### ISSUE TITLE
Redo Age Comp Prior to improve convergence speed

### ISSUE THREAD

**Issue by tbuehrens** [casruff/Skagit-River-Steelhead-Forecast#12]
@casruff after inspecting more summaries, it is clear that the hierarchical dirichlet prior is causing converge issues, particularly for rare age classes. The good news is the effect on the overall model is miniscule because age 8 and age 3 fish (age classes 1 and 6) are really rare. An alternative parameterization could be in terms of the additive log odds ratio:

here, lorc = "log odds ratio component", orc = odds ratio component, and p = proportions by age class...softmax is a function that does the calcs more efficiently but doesn't exist in jags...this would be an easy improvement to implement...Eric, Mark, and I have all started using it independently because it tends to converge better than the dirichlet
```
for(a in 1:A){
    lorc[a]~normal(0,3)
    orc[a] = exp(lorc[a])
    p[a] = orc[a] /(sum(orc[a]))
}

#p[1:A] = softmax(lorc[1:A]) #if softmax existed in JAGS


```

**Comment by tbuehrens** [comment]
@casruff wondering it switching age comp to a multivariate logit normal random walk would be even better...could it be that time varying age comp of recruits accounts for some of the forecast misses recently?

**Comment by tbuehrens** [comment]
logit normal random walk is done...could update previous models to be logit normal random effects...likely would have negligible effects on estimates from original models but speed convergence
